spec/features/work_packages/new_work_package_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+require 'features/work_packages/details/inplace_editor/work_package_field'
+require 'features/work_packages/work_packages_page'
+
+describe 'new work package', js: true, retry: 3, retry_wait: 5 do
```

Comment:
Did you configure this somewhere? `spec/support/rspec_retry` does nothing on that end, and the gem's default is retry once, without delay iirc.